services/src/api/handlers/datasets.rs

```diff
@@ -1473,24 +1289,111 @@ pub async fn list_volume_file_layers_handler<C: ApplicationContext>(
     Ok(web::Json(VolumeFileLayersResponse { layers }))
 }
 
+/// Creates a new dataset using files available on the volumes.
+/// The created dataset will be visible to all users.
+/// Requires an admin session.
```

Comment:
the description does not make sense.

previously it was "Creates a new dataset referencing files. Users can reference previously uploaded files. Admins can reference files from a volume"

Reply:
[c29186e](https://github.com/geo-engine/geoengine/pull/1006/commits/c29186e758ba3ceb8168346fef331a2e98623436)